sdks/java/core/src/main/java/org/apache/beam/sdk/values/PCollection.java

```diff
@@ -73,14 +68,11 @@
    */
   public enum IsBounded {
     /**
-     * Indicates that a {@link PCollection} contains bounded data elements, such as
-     * {@link PCollection PCollections} from {@link TextIO}, {@link BigQueryIO},
-     * {@link Create} e.t.c.
+     * Indicates that a {@link PCollection} contains a bounded number of elements.
      */
     BOUNDED,
     /**
-     * Indicates that a {@link PCollection} contains unbounded data elements, such as
-     * {@link PCollection PCollections} from {@link PubsubIO}.
+     * Indicates that a {@link PCollection} contains unboundedly many data elements.
```

Comment:
contains unboundedly many data elements -> an unbounded number of elements

just mirror the BOUNDED javadoc